### DISCUSSION TITLE
How can I calculate height/ width or lenght of a LINE

### DISCUSSION THREAD

**Discussion by robertwilli1** [pikasojs/pikaso#54] [Q&A] [answered]
Sir, I want to calculate the length of a line.
I have calculated the height width of shapes like rectangle in pikaso but when it come to displaying length of a Line, I can't find any attribute like width height in "data.shapes[0].node.attrs" because line has points:[50,50,300,50] 
So how can I display length of Line

**Comment by raminious** [comment, marked as answer]
@osamatan125 I think you can use `getClientRect`  
so it would be `data.shapes[0].node.getClientRect()`

more info [https://konvajs.org/api/Konva.Line.html#getClientRect](https://konvajs.org/api/Konva.Line.html#getClientRect)

**Reply by robertwilli1** [reply]
That is awesome man, thanks.
